gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -670,42 +670,55 @@
     [150180, 'array_slice', 'ARRAY_DECIMALV2', ['ARRAY_DECIMALV2', 'BIGINT', 'BIGINT'], 'ArrayFunctions::array_slice_decimalv2'],
     [150181, 'array_slice', 'ARRAY_VARCHAR',   ['ARRAY_VARCHAR', 'BIGINT', 'BIGINT'],   'ArrayFunctions::array_slice_varchar'],
 
-    [150190, 'array_concat', 'ARRAY_DATE',      ['ARRAY_DATE', "..."],      'ArrayFunctions::array_concat_date'],
```

Comment:
Why change the id, will there be compatibility problems?

Reply:
> Why change the id, will there be compatibility problems?

Just let all array_slices together. But maybe have compatibility problems caused by Published FE, so let me change this.